client/src/main/scripts/components/VideoLite.tsx

```diff
@@ -75,22 +73,22 @@ class VideoLite extends React.Component<VideoLiteProps, VideoLiteState> {
     };
   }
   cueVideo = (props: VideoLiteProps) => {
-    if (this.player) {
-      if (props.clipRange) {
-        this.player.cueVideoById({
-          videoId: props.videoId,
-          startSeconds: props.clipRange[0],
-          endSeconds: props.clipRange[1],
-          suggestedQuality: "default",
-        });
-        this.playerVars.start = props.clipRange[0];
-        this.playerVars.end = props.clipRange[1];
-      } else {
-        this.player.cueVideoById({
-          videoId: props.videoId,
-          suggestedQuality: "default",
-        });
-      }
+    if (props.clipRange) {
```

Comment:
How could there be a video, but not a clipRange? If that never happens, we shouldn't support it.  And if it does happen, what does it mean to have a video but no part in particular to play?

Reply:
Prior to n-gram this was helpful so that we could cue a video before a user had chosen a clip. This would make the video experience quicker and smoother once a user had chosen a clip. Now, however, we don't render this component unless there's a clip, so this is valid.

Reply:
Prior to n-gram, we would choose the video of the first search result, correct?  So we might as well also choose the first clip?

In general, there are some cases where an array could be empty.  But usually that's a special case which must be handled in a unique way, and every other case requires the array to be non empty.  For the non-empty case, using the first value in the array as a "default" value is a useful general-purpose technique for reducing your cyclomatic complexity.